Servers/Zeus/ARC/.arc/settings.py

```diff
@@ -0,0 +1,51 @@
+
+"""
+ARC's settings
+"""
+
+import os
+import string
+
+
+servers = {
+    'local': {  # Each Zeus node containes 80 cores and 378 GB RAM
+        'cluster_soft': 'PBS',
+        'un': 'alon',
```

Comment:
Maybe in the places users need to fill up their own info, you add something like a variable.

Reply:
The server knows your the username (`$USER`), sometimes in ARC we use that. I'm not sure it is used throughout. Once we fix that in ARC we can remove the 'un' here altogether

Reply:
Ok, so could you just comment so that people will now to replace it?